samples/PactV3/EventApi/Consumer/DateTimeFactory.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Consumer
+{
+    public class DateTimeFactory
```

Comment:
This is a bit weird that you overwrite the `Now` function in the tests. Usually you'd have an `IDateTimeProvider` and then you can have real and mock implementation of it.

Reply:
I think this is part of the original API sample app